src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -111,14 +112,24 @@ public static String getPagesPackage() {
      * @return the timeOut
      */
     public static int getTimeOut() {
+        if (TIMEOUT == null) {
+            if (Props.get("page.load.timeout") == null || "".equals(Props.get("page.load.timeout"))) {
```

Comment:
Can we extract Props.get("page.load.timeout") into a local variable instead of calling it dirrectly?